dev/CommonStyles/TextBox_themeresources.xaml

```diff
@@ -159,17 +159,17 @@
                     <ScaleTransform ScaleY="-1" CenterY="0.5"/>
                 </LinearGradientBrush.RelativeTransform>
                 <LinearGradientBrush.GradientStops>
-                    <GradientStop Offset="0" Color="{StaticResource ControlAAFillColorDefault}"/>
+                    <GradientStop Offset="0.5" Color="{StaticResource ControlAAStrokeColorDefault}"/>
                     <GradientStop Offset="1.0" Color="{StaticResource ControlStrokeColorDefault}"/>
                 </LinearGradientBrush.GradientStops>
             </LinearGradientBrush>
 
-            <LinearGradientBrush x:Key="TextControlElevationBorderFocusedBrush" MappingMode="Absolute" StartPoint="0,1.5" EndPoint="0,2.5">
+            <LinearGradientBrush x:Key="TextControlElevationBorderFocusedBrush" MappingMode="Absolute" StartPoint="0,0" EndPoint="0,2">
                 <LinearGradientBrush.RelativeTransform>
                     <ScaleTransform ScaleY="-1" CenterY="0.5"/>
                 </LinearGradientBrush.RelativeTransform>
                 <LinearGradientBrush.GradientStops>
-                    <GradientStop Offset="0" Color="{ThemeResource SystemAccentColorDark1}"/>
+                    <GradientStop Offset="1.0" Color="{ThemeResource SystemAccentColorDark1}"/>
```

Comment:
Two stops at the same coordinate seems odd, especially when there's not a stop at 0. What does this mean?

Reply:
@jevansaks 
`<LinearGradientBrush x:Key="TextControlElevationBorderFocusedBrush" MappingMode="Absolute" StartPoint="0,0" EndPoint="0,2">
                <LinearGradientBrush.RelativeTransform>
                    <ScaleTransform ScaleY="-1" CenterY="0.5"/>
                </LinearGradientBrush.RelativeTransform>
                <LinearGradientBrush.GradientStops>
                    <GradientStop Offset="1.0" Color="{ThemeResource SystemAccentColorDark1}"/>
                    <GradientStop Offset="1.0" Color="{StaticResource ControlStrokeColorDefault}"/>
                </LinearGradientBrush.GradientStops>
            </LinearGradientBrush>
`
As I understand it: here we have **StartPoint=0,0** and **EndPoint=0,2**; by setting both stops to 1 we get the bottom stroke of the border to be one solid color (in this case SystemAccentColorDark1) and not interpolate between SystemAccentColorDark1 and ControlStrokeColorDefault at all until we reach end point 0,2. After we reach that point (0,2) we'll switch the color to SystemAccentColorDark1. By doing this, we'll make sure that bottom stroke is one solid color and  is not interpolated along the way until we reach the stop (it will interpolate anyway very very close to the edge of the bottom stroke):
![image](https://user-images.githubusercontent.com/21356912/108445240-e6134d80-7210-11eb-8e26-9f6f424458b3.png)